src/components/manage/Blocks/Listing/SummaryTemplate.jsx

```diff
@@ -26,13 +27,20 @@ const SummaryTemplate = ({ items, linkTitle, linkHref, isEditMode }) => {
         {items.map((item) => (
           <div className="listing-item" key={item['@id']}>
             <ConditionalLink item={item} condition={!isEditMode}>
-              {!item.image_field && <img src={DefaultImageSVG} alt="" />}
+              {!item.image_field && (
```

Comment:
Wondering if the Image component could take care of the default image as well, passing a prop with the default image to use and use it if the image is not present.

Reply:
Maybe do it as part of PreviewImage from here? https://github.com/plone/volto/pull/2888

Or maybe push that functionality in this PR?